dali/python/backend_impl.cc

```diff
@@ -70,6 +70,19 @@ static string TensorLayoutRepr(const TensorLayout &tl) {
   return ss.str();
 }
 
+template<typename Backend>
+py::dict ArrayInterfaceRepr(Tensor<Backend> &t) {
+  py::dict d;
+  py::tuple tup(2);
+  d["typestr"] = FormatStrFromType(t.type());
+  // __array_interface__ expects shape to be a tuple
+  d["shape"] = py::tuple(py_shape<Backend>(t));
+  tup[0] = py::reinterpret_borrow<py::object>(PyLong_FromVoidPtr(t.raw_mutable_data()));
+  tup[1] = true;
```

Comment:
I would maybe add a comment that explains that this is indicating that the data is read only.

Reply:
Done